### ISSUE TITLE
Admonitions in included markdown pages are not rendered

### ISSUE THREAD

**Issue by amorison** [Fortran-FOSS-Programmers/ford#665]
When including another markdown document using the `markdown_include` extension, admonitions (`@note`, `@warning`, etc) are not rendered in the produced document but simply left as is.

For example, with the following `README.md`

```md
My project
=========

Description of what it does.

@note
Important thing to keep in mind.
@endnote
```

and with the following inclusion in the documentation itself

```md
{!path/to/README.md!}
```

the note is not rendered.

I am not sure whether this is a bug with the admonition extension as implemented in Ford or something else. Other extensions (such as math expression rendering) seem to work, so this puzzles me.

**Comment by d7919** [comment]
I think this is due to the priority number associated with the `markdown_include` and FORD specific Admonition extensions. The former uses a priority of 101 whilst the latter has 105. As larger priority number extensions are run before lower priority ones, this means we don't include files until after we've parsed for admonitions. Dropping admonition to 101 (equal to markdown_include) seems to fix the issue for a local project which showed the same issue.

**Comment by amorison** [comment]
I confirm that your fix works for me too, thanks!

**Comment by ZedThree** [comment]
Fixed in #666
